src/deps.txt

```diff
@@ -23,9 +23,8 @@ selinux-policy-targeted rpm-build
 # Standard build tools
 make git rpm-build
 
-# virt-install dependencies
-libvirt libguestfs-tools /usr/bin/qemu-img /usr/bin/virsh /usr/bin/virt-install
-qemu-kvm
+# virt dependencies
+libguestfs-tools /usr/bin/qemu-img qemu-kvm
```

Comment:
This change I didn't test.  We may actually need more libvirt bits.

Reply:
I think this is okay. Most of our uses `LIBGUESTFS_BACKEND=direct` anyway.

Reply:
It turns out `libguestfs` depends on `libvirt-daemon-kvm >= 0.10.2-3` so we're going to get it anyways.